### ISSUE TITLE
Custom selectors that are vendor specific

### ISSUE THREAD

**Issue by martynchamberlin** [casesandberg/reactcss#133]
Love this `reactcss` project and what it's seeking to do. I have a `less` file in a React project that I'm trying to convert to `reactcss` and ran into this line that I have in my `less` file: 

```
-ms-overflow-style: none;
```

How would I translate that to `reactcss`? I tried this, which I got as an idea from [this comment](https://github.com/facebook/react/issues/2020#issuecomment-123240454), but no luck. 

```
const styles = reactCSS({
  'default': {
    sidebar: {
      overflowStyle: 'none; -ms-overflow-style: none',
    }
  }
});
```

Any ideas? 

At a higher level, what I'm wondering is, how can we have custom selectors and selectors that are vendor specific? Is that possible?

**Comment by casesandberg** [comment]
Hey @martynchamberlin! You can read more about vendor prefixes in react [here](http://stackoverflow.com/a/32100497/989006).

TL;DR Drop the leading dash and uppercase the vendor namespace:
```
const styles = reactCSS({
  'default': {
    sidebar: {
      overflowStyle: 'none',
      MsOverflowStyle: 'none'
    }
  }
});
```

**Comment by martynchamberlin** [comment]
Ah, I see, right. Thanks for clarifying!
